js/demo/lazy-loading.js

```diff
@@ -105,8 +93,8 @@ var EffecktLazyLoading = {
         self._scrollPage();
       }, 1000 / 60);
     }
-  },
+  }
```

Comment:
Let's not handle this kind of things in unrelated commits. We can cleanup all the things when handling https://github.com/h5bp/Effeckt.css/issues/314.